src/DurableTask.SqlServer.AzureFunctions/SqlTargetScaler.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace DurableTask.SqlServer.AzureFunctions
+{
+    using System.Threading.Tasks;
+    using Microsoft.Azure.WebJobs.Host.Scale;
+
+#if !NETSTANDARD
+    public class SqlTargetScaler : ITargetScaler
+    {
+        readonly SqlMetricsProvider sqlMetricsProvider;
+        readonly TargetScalerResult scaleResult;
```

Comment:
Why do we cache the `TargetScalerResult` object instead of creating a new one for each call?

Reply:
I went ahead and removed this caching. I'm worried about potential race condition side effects that might happen with shared state.